test/functional/wallet-hd.py

```diff
@@ -78,7 +78,7 @@ def run_test (self):
         shutil.rmtree(os.path.join(tmpdir, "node1/regtest/chainstate"))
         shutil.rmtree(os.path.join(tmpdir, "node1/regtest/evodb"))
         shutil.rmtree(os.path.join(tmpdir, "node1/regtest/llmq"))
-        shutil.copyfile(os.path.join(tmpdir, "hd.bak"), os.path.join(tmpdir, "node1/regtest/wallet.dat"))
+        shutil.copyfile(os.path.join(tmpdir, "hd.bak"), os.path.join(tmpdir, "node1/regtest/wallets/wallet.dat"))
```

Comment:
Should also update line 106 cause otherwise we test nothing there - `hd.bak` won't replace the wallet we load on start (can easily verify this by adding `os.remove(tmpdir + "/node1/regtest/wallets/wallet.dat")` line right before it - it's going to fail without the fix). Looks like this they missed it in btc and fixed it later in 12076 https://github.com/bitcoin/bitcoin/commit/c8330d4216e9f8e31c4e910a6bdf42f09458b2cd#diff-9ef61f3ae1a87f72e3ea5cfd6a004110R99

Reply:
see https://github.com/dashpay/dash/pull/3324/commits/9609598ae792b19ab208418138ba5121f9f95f33